scripts/gentest/src/main.rs

```diff
@@ -253,21 +259,31 @@ fn generate_assertions(ident: &str, node: &Value, use_rounding: bool) -> TokenSt
 
     if use_rounding {
         quote!(
-            let Layout { size, location, .. } = taffy.layout(#ident).unwrap();
+            #[cfg_attr(not(feature = "content_size"), allow(unused_variables))]
+            let layout @ Layout { size, location, .. } = taffy.layout(#ident).unwrap();
             assert_eq!(size.width, #width, "width of node {:?}. Expected {}. Actual {}", #ident,  #width, size.width);
             assert_eq!(size.height, #height, "height of node {:?}. Expected {}. Actual {}", #ident,  #height, size.height);
             assert_eq!(location.x, #x, "x of node {:?}. Expected {}. Actual {}", #ident,  #x, location.x);
             assert_eq!(location.y, #y, "y of node {:?}. Expected {}. Actual {}", #ident,  #y, location.y);
+            #[cfg(feature = "content_size")]
+            assert_eq!(layout.scroll_width(), #scroll_width, "scroll_width of node {:?}. Expected {}. Actual {}", #ident,  #scroll_width, layout.scroll_width());
+            #[cfg(feature = "content_size")]
+            assert_eq!(layout.scroll_height(), #scroll_height, "scroll_height of node {:?}. Expected {}. Actual {}", #ident,  #scroll_height, layout.scroll_height());
 
             #children
         )
     } else {
         quote!(
-            let Layout { size, location, .. } = taffy.layout(#ident).unwrap();
+            #[cfg_attr(not(feature = "content_size"), allow(unused_variables))]
+            let layout @ Layout { size, location, .. } = taffy.layout(#ident).unwrap();
             assert!(size.width - #width < 0.1, "width of node {:?}. Expected {}. Actual {}", #ident,  #width, size.width);
             assert!(size.height - #height < 0.1, "height of node {:?}. Expected {}. Actual {}", #ident,  #height, size.height);
             assert!(location.x - #x < 0.1, "x of node {:?}. Expected {}. Actual {}", #ident,  #x, location.x);
             assert!(location.y - #y < 0.1, "y of node {:?}. Expected {}. Actual {}", #ident,  #y, location.y);
+            #[cfg(feature = "content_size")]
+            assert!(layout.scroll_width() - #scroll_width < 0.1, "scroll_width of node {:?}. Expected {}. Actual {}", #ident,  #scroll_width, layout.scroll_width());
```

Comment:
It would also be nice to pull out 0.1 into a shared epsilon value for clarity.

Reply:
Hmm... yeah this almost certainly ought to be comparing absolute difference. For the other values above too. I guess this wasn't noticed because it's working in practice (I think most tests are using the rounded values anyway, it's only a handful that are using the unrounded values). I believe the scroll values might able to use exact equality in both cases because we don't have access to the the float values of the scroll width/height anyway.

Reply:
These have been updated to use `.abs()`.